src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -1503,10 +1530,10 @@ public static boolean calledByAnIncompleteSyntheticClass(MethodCallExpr method)
    * @param missedClass the class to be updated
    */
   public void updateMissingClass(UnsolvedClass missedClass) {
+    String qualifiedName = missedClass.getPackageName() + "." + missedClass.getClassName();
```

Comment:
This should probably be a method in `UnsolvedClass` (i.e., this should be a call to a new method `UnsolvedClass#getQualifiedName()`.